src/controllers/ControllerTasks.js

```diff
@@ -126,22 +126,38 @@ module.exports.createTasks = function(request, reply) {
 
         q.defer(function(cb) {
           // save data on type
-          client.bulk({
-            maxRetries: 5,
-            index: 'tofix',
-            id: task.idtask,
-            type: task.idtask,
-            body: bulk
-          }, function(err, resp) {
-            if (err) {
-              console.log(err);
-            } else {
-              console.log(resp.items);
-              cb();
-            }
-          });
-        });
+          var bulkChunks = _.chunk(bulk, 2000);
+          var counter = 0;
+          var errorsCounter = 0;
 
+          function saveData(bulkChunk) {
+            client.bulk({
+              maxRetries: 5,
+              index: 'tofix',
+              id: task.idtask,
+              type: task.idtask,
+              body: bulkChunk
+            }, function(err) {
+              if (err) {
+                //try 3 times to save the chunk
+                errorsCounter++;
+                if (errorsCounter === 3) {
```

Comment:
@Rub21 Why are we expecting it to succeed on a retry after failure? Is this how ElasticSearch works?